src/styles/tasks.module.scss

```diff
@@ -1,13 +1,27 @@
 $primaryColor: #041484;
 $disabledColor: gray;
 
+.tasksContainer {
+    width: 64%;
+    @media (max-width: 1280px) {
+        width: 85%;
+    }
+    @media (max-width: 768px) {
+        width: 90%;
+    }
+    @media (max-width: 600px) {
+        width: 100%;
+    }
```

Comment:
why cant we follow the proper rule for media queries, why cant we separate out the media query instead of crowding everything into one place?